src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
```

Comment:
use justify-content property on header to make a distance between the logo and nav

Reply:
laos use aligh-items to center logo and nav vertically